File: reni-backend/src/main/java/pe/gob/mimp/reni/dto/response/EstructuraBuscarResponse.java
package pe.gob.mimp.reni.dto.response;

import java.io.Serializable;
import java.util.List;

import lombok.Getter;
import lombok.ToString;
import lombok.Setter;

@Getter
@Setter
@ToString
public class EstructuraBuscarResponse implements Serializable {
	private static final long serialVersionUID = 1L;

	private String nombre;
	private String descripcion;
	private String idTipoUsuario;
	private Long idServicio;
	private String nomServicio;
	private Long idEntidad;
	private String nomEntidad;
	private Long idLineaInter;
	private String nomLineaInter;
	private Long idEstructuraRepl;
	private String nomEstructuraRepl;
	private List<EstructuraBuscarParametroResponse> listaParametro;
}

File: reni-frontend/src/main/reni-frontend/src/app/modules/intranet/dto/response/estructura-buscar.response.ts
import { EstructuraBuscarParametroResponse } from "./estructura-buscar-parametro.response";

export class EstructuraBuscarResponse {
    nombre: string;
    descripcion: string;
    idTipoUsuario: string;
    idServicio: number;
    nomServicio: string;
    idEntidad: number;
    nomEntidad: string;
    idLineaInter: number;
    nomLineaInter: string;
    idEstructuraRepl: number;
    nomEstructuraRepl: string;
    listaParametro: EstructuraBuscarParametroResponse[];
}